src/utils/urls.ts

```diff
@@ -17,7 +17,7 @@ export const urlPageMdEdit = (slug: string) =>
 export const urlArticleMdEdit = (year: string, slug: string) =>
   externalTo('github', join(REPOSITORY_PATH, 'edit', 'master', '_articles', 'markdowns', year, `${slug}.md`))
 
-export const mailTo = (to: string) => `mailto:${to}`
+export const mailTo = (to: string) => join('mailto:', to)
```

Comment:
これは `mailto:/hoge@example.com` になってヤバい
```suggestion
export const mailTo = (to: string) => `mailto:${to}`
```